### PR TITLE
Iris via changes

### PR DESCRIPTION
**Pull request by nooges**
## Description

Change PRODUCT_ID of Iris boards to allow for detection of different versions in VIA.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by noroadsleft** [comment]
I want it on record that I absolutely hate the repetition of the Manufacturer string inside the Product string.

File: keyboards/keebio/iris/rev2/config.h [hunk 1]
```diff
@@ -19,10 +19,10 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 /* USB Device descriptor parameter */
 #define VENDOR_ID       0xCB10
-#define PRODUCT_ID      0x1256
+#define PRODUCT_ID      0x2256
 #define DEVICE_VER      0x0200
 #define MANUFACTURER    Keebio
-#define PRODUCT         Iris Keyboard
+#define PRODUCT         Keebio Iris Rev. 2
```
Comment (noroadsleft) [inline keyboards/keebio/iris/rev2/config.h]: I'd like it on record that I absolutely hate and am vehemently opposed to repeating the Manufacturer name inside the Product string.

Comment (Erovia) [reply keyboards/keebio/iris/rev2/config.h]: I agree with @noroadsleft, it looks absolutely stupid on anything but Windows.

Comment (nooges) [reply keyboards/keebio/iris/rev2/config.h]: Oh I know, just being a VIA shill I guess: https://caniusevia.com/docs/configuring_qmk#change-product

**Pull request merged**